tests/test_cli.py

```diff
@@ -653,6 +653,8 @@ def add_compare(self, cat, args, compbase, **kwargs):
 --network user,address.type=ccw,address.cssid=0xfe,address.ssid=0,address.devno=01,boot.order=15,boot.loadparm=SYSTEM1
 --network model=vmxnet3
 --network backend.type=passt,backend.logFile=/tmp/foo.log,portForward0.proto=tcp,portForward0.address=192.168.10.10,portForward0.dev=eth0,portForward0.range0.start=4000,portForward0.range0.end=5000,portForward0.range0.to=10000,portForward0.range0.exclude=no,portForward0.range1.start=6000,portForward1.proto=tcp,portForward1.range0.start=2022,portForward1.range0.to=22
+--network passt,portForward=8080:80
+--network passt,portForward0=7000-8000/udp,portForward1=127.0.0.1:2222:22
```

Comment:
This could probably use a bit more coverage, especially for the less common combinations. For example:

```
--network passt,portForward0=2001:db8:ac10:fd01::1:10:3000-4000:30,portForward1=127.0.0.1:5000-6000:50
```

We're also missing coverage for the very simplest case, such as:

```
--network passt,portForward=8080
```

It all seems to work fine right now! I'm just concerned that it might accidentally get broken in the future.

Reply:
ACK, I think it looks good now